### PR TITLE
Task3

### PR DESCRIPTION
**Pull request by Vanad1um**

### CODE REVIEW

File: ui/index.html [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+            <img src="images\phone.png" class="phone">
+            <div class="login_phone_num">info: +375-XX-XXX-XX-XX</div>
+
+            <div class="login_button">
+                <div class="login_text_style login_user_you">You:</div>
+                <div class="login_text_style login_user_name">SushiMax</div>
+                <div class="login_line1"></div>
+                <div class="login_text_style login_add_post">Add Post</div>
+                <div class="login_line2"></div>
+                <div class="login_text_style login_log_out">Log Out</div>
```
Comment (YauheniyaBasaranovich) [inline ui/index.html:34]: Should be `button` or `a`, not `div`

File: ui/index.html [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+
+        <div class="login_window">
+            <img src="images\phone.png" class="phone">
+            <div class="login_phone_num">info: +375-XX-XXX-XX-XX</div>
+
+            <div class="login_button">
+                <div class="login_text_style login_user_you">You:</div>
+                <div class="login_text_style login_user_name">SushiMax</div>
+                <div class="login_line1"></div>
+                <div class="login_text_style login_add_post">Add Post</div>
```
Comment (YauheniyaBasaranovich) [inline ui/index.html:32]: Please use semantic tags, eg `button` here.

File: ui/index.html [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+                <div class="login_line2"></div>
+                <div class="login_text_style login_log_out">Log Out</div>
+            </div>
+        </div>
+
+        <div class="header_menu">
+
```
Comment (YauheniyaBasaranovich) [inline ui/index.html:39]: Do you need this empty element?

File: ui/index.html [hunk 4]
```diff
@@ -0,0 +1,441 @@
...
+        </div>
+    </div>
+</header>
+
+<body>
+<div class="main">
```
Comment (YauheniyaBasaranovich) [inline ui/index.html:45]: Why it's outside of the body? It's supposed to be inside

File: ui/index.html [hunk 5]
```diff
@@ -0,0 +1,441 @@
...
+    <div class="side_panel">
+        <div class="side_date_text">Date</div>
```
Comment (YauheniyaBasaranovich) [inline ui/index.html:47]: Pls use corresponding input controls

File: ui/index.html [hunk 6]
```diff
@@ -0,0 +1,441 @@
...
+        <div class="side_search_from_field"></div>
+        <div class="side_search_to_field"></div>
+        <div class="side_vendors_name_text">Vendor's name</div>
+        <div class="side_vendors_field"></div>
+
+        <ul class="side_hashtag_list1">
+            <li class="description_style">#food</li>
```
Comment (YauheniyaBasaranovich) [inline ui/index.html:54]: I think it would be better if you added `a` tag inside of `li`

File: ui/index.html [hunk 7]
```diff
@@ -0,0 +1,441 @@
...
+        <li class="content_offer">
+            <img class="offer_pic" src="images\preview_sushi.png">
+            <p class="offer_name_style offer_name_text">SuperSushi Set “McDonalds”</p>
+            <div class="hashtags_style hashtags_text">#sushi #food</div>
+            <div class="description_style description_text">Sushi, a staple rice dish of Japanese cuisine,
+                consisting of cooked
+                rice flavoured with vinegar and a variety of vegetable, egg, or raw seafood garnishes and served
+                cold.
+            </div>
+            <img class="star_rating_first" src="images\star_yellow.png">
```
Comment (YauheniyaBasaranovich) [inline ui/index.html:93]: I think it's better to wrap all star element to common container to keep markup clear

File: ui/index.html [hunk 8]
```diff
@@ -0,0 +1,441 @@
...
+            <img class="star_rating_third" src="images\star_yellow.png">
+            <img class="star_rating_fourth" src="images\star_yellow.png">
+            <img class="star_rating_fifth" src="images\star_grey.png">
+            <div class="description_style reviews_num_text">7 reviews</div>
+            <div class="description_style post_review_text">Post Your Review</div>
+            <div class="hashtags_style vendors_name_text">SushiMax</div>
+            <div class="offer_date_post_text">19.02.2021 18:59</div>
+
+            <div class="delete_offer_button">
+                <P class="delete_offer_button_text right_buttons_text_style">Delete</P>
```
Comment (YauheniyaBasaranovich) [inline ui/index.html:104]: Please always write HTML tags in lower case

File: ui/styles.css [hunk 9]
```diff
@@ -0,0 +1,711 @@
+*{
+    padding: 0;
+    margin: 0;
+}
+
+a {
+text-decoration: none;
```
Comment (YauheniyaBasaranovich) [inline ui/styles.css:7]: Please align your css code.

**Review by YauheniyaBasaranovich — APPROVED** [review]
3 weeks late = -1.5 points;
Not responsive = -2 points;
Non-semantic markup + other issues = -1.5 points.

So, your score for now is 5, fix my comments if you need higher one.

File: ui/styles.css (hunk 9 above)
Comment (Vanad1um) [reply ui/styles.css:7]: Thanks, i'll try to make some changes in next week.

File: ui/index.html [hunk 10]
```diff
@@ -0,0 +1,441 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Главная - BFsales</title>
+    <link href="css\styles.css" rel="stylesheet">
```
[the remark this answers is not in the document]
Comment (Vanad1um) [reply ui/index.html:6]: It is in a folder of project i think.

File: ui/index.html [hunk 11]
```diff
@@ -0,0 +1,441 @@
...
+    <link rel="SHORTCUT ICON" href="images\favicon.ico" type="image\x-icon">
```
[the remark this answers is not in the document]
Comment (Vanad1um) [reply ui/index.html:7]: I will add it, i misunderstood requirements for task folder.

File: ui/index.html (hunk 3 above)
Comment (Vanad1um) [reply ui/index.html:39]: I need it, it is just a design feature

**Review by YauheniyaBasaranovich — COMMENTED** [review]
Better now, but still not fully responsive.

**Pull request merged**